app/controllers/cloud_object_store_container_controller.rb

```diff
@@ -6,17 +6,17 @@ class CloudObjectStoreContainerController < ApplicationController
 
   include Mixins::GenericListMixin
   include Mixins::GenericShowMixin
+  include Mixins::GenericButtonMixin
   include Mixins::GenericSessionMixin
 
   def breadcrumb_name(_model)
     ui_lookup(:tables => "cloud_object_store_container")
   end
 
-  # handle buttons pressed on the button bar
   def button
```

Comment:
I am not sure that adding a method for these simple one-line statemants helps to improve the code.

At least for me the readability of the code goes down. I have to find the method and read the line elsewhere.